astronomer/providers/google/cloud/hooks/bigquery.py

```diff
@@ -4,10 +4,10 @@
 from airflow.exceptions import AirflowException
 from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook
 from gcloud.aio.bigquery import Job, Table
-from google.cloud.bigquery import CopyJob, ExtractJob, LoadJob, QueryJob
 from requests import Session
 
 from astronomer.providers.google.common.hooks.base_google import GoogleBaseHookAsync
+from google.cloud.bigquery import CopyJob, ExtractJob, LoadJob, QueryJob
```

Comment:
this should stay at the top -- it is unfortunately detecting this is a "internal" module hence it is at the bottom -- check how we did it on Airflow https://github.com/apache/airflow/blob/d03b9a76914babef23b84acdb6061b0b93bdc2e3/pyproject.toml#L75-L97 -- we should do the same here

Reply:
Ah, didn't notice. thanks for pointing out